File: JAVA/Daily Flash Codes/Week1/Day3_Jan15/code_java/program3.java
/*
Program 3: Write a Program to take number of month and print the number of
days in that month.
Input: 4
Output: April is a 30 days’ month.
 */


import java.io.*;

class Program {
	public static void main(String args[]) throws IOException {
		
		int  i;

	do {
		System.out.println("Enter Month num: ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in) );
		i = Integer.parseInt(br.readLine());	//to read integerr

		if(i < 1 || i > 12)
			System.out.println("Wrong input !! Try again ");
	}while(i <1 || i > 12);

	switch(i) {
		case 1:
			System.out.println("January is a 31 days' month");
		        break;	
		case 2:
			System.out.println("February is a 28/29 days' month" );
		        break;	

		case 3:
			System.out.println("March is a 31 days' month");
		        break;	
		case 4:
			System.out.println("April is a 30 days' month");
		        break;	
		case 5:
			System.out.println("may is a 31 days' month");
		        break;	
		case 6:
			System.out.println("June is a 30 days' month");
		        break;	
		case 7:
			System.out.println("July is a 31 days' month");
		        break;	
		case 8:
			System.out.println("August is a 31 days' month");
		        break;	
		case 9:
			System.out.println("September is a 30 days' month");
		        break;	
		case 10:
			System.out.println("October is a 31 days' month");
		        break;	
		case 11:
			System.out.println("November is a 30 days' month");
		        break;	
		case 12:
			System.out.println("December is a 31 days' month");
		        break;	
				
	}
	}
}

File: C++/Daily Flash/week1/Day3_Jan15/code_C++/program3.cpp
/*
 *
Program 3: Write a Program to take number of month and print the number of days in that month.
Input: 4
Output: April is a 30 days’ month.
*
*/

#include <iostream>


int main() {
	int  i;

	do {
		std::cout << "Enter number of month(1-12): ";
		std::cin >> i;

		if(i < 1 || i > 12)
			std::cout << "Wrong input !! Try again " <<std::endl;
	}while(i < 1 || i > 12);

	switch(i) {
		case 1:
			std::cout << "January is a 31 days' month" <<std::endl;
		        break;	
		case 2:
			std::cout << "February is a 28/29 days' month" <<std::endl;
		        break;	

		case 3:
			std::cout << "March is a 31 days' month" <<std::endl;
		        break;	
		case 4:
			std::cout << "April is a 30 days' month" <<std::endl;
		        break;	
		case 5:
			std::cout << "may is a 31 days' month" <<std::endl;
		        break;	
		case 6:
			std::cout << "June is a 30 days' month" <<std::endl;
		        break;	
		case 7:
			std::cout << "July is a 31 days' month" <<std::endl;
		        break;	
		case 8:
			std::cout << "August is a 31 days' month" <<std::endl;
		        break;	
		case 9:
			std::cout << "September is a 30 days' month" <<std::endl;
		        break;	
		case 10:
			std::cout << "October is a 31 days' month" <<std::endl;
		        break;	
		case 11:
			std::cout << "November is a 30 days' month" <<std::endl;
		        break;	
		case 12:
			std::cout << "December is a 31 days' month" <<std::endl;
		        break;	
				
	}


}
